### DISCUSSION TITLE
XP-PEN Deco 02 not working at all

### DISCUSSION THREAD

**Discussion by Vulkanodox** [xournalpp/xournalpp#3636] [Q&A] [answered]
Hi, 

I'm looking for some competent software to take notes and create quick visualizations and xournal++ seems competent but my Deco 02 is not working. I'm on windows.

It is recognized by the software:
![image](https://user-images.githubusercontent.com/65197406/144667615-b62b5d20-4c76-432f-b8af-5f42f3f94f96.png)
 but does not work at all.

if I activate the mouse input it works indirectly because the stylus moves the mouse which then is used as input for xournal++ but all Pen features such as pressure sensitivity don't work. 

It is not a hardware issue since the driver for the tablet and stylus show the pressure and other programs like gimp can pick it up. 

And again nothing works, not even drawing.

Any pointers or open a bug?

**Comment by ewlhas0** [comment]
I too have the XP-Pen Deco 02, mine works fine. (Win 10) It is detected fine. Could you screenshot the 'Stylus' tab, as you can set the handling of pressure sensitivity input there.

**Reply by Vulkanodox** [reply]
![image](https://user-images.githubusercontent.com/65197406/145464137-eb78be8c-0219-4500-98c7-aa19a1ea107b.png)
cant change the language but should be the same layout of settings.

windows ink doesn't change it

**Reply by ewlhas0** [reply]
I meant the 'Stift'/'Stylus' Tab in the Xournal++ settings. (I speak german as well)
There you can set the behavior af the stylus as you like, I attached a screenshot (I don't use pressure detection for writing)


![sc1](https://user-images.githubusercontent.com/95862384/145554887-4ccce776-a80c-42d8-a86d-a17fc8a37a20.PNG)



for anyone not speaking german:

![sc2](https://user-images.githubusercontent.com/95862384/145555199-90019493-61ee-4132-8642-edbbe2b2f1c7.JPG)

**Reply by Vulkanodox** [reply]
![image](https://user-images.githubusercontent.com/65197406/145583064-fea90abb-6ebd-432a-a922-ae249ab4d889.png)
already tried to change the sensitivity scales with no effect. 

do you have the mouse input turned on?

![image](https://user-images.githubusercontent.com/65197406/145583206-25ce0caf-1f4c-4bb7-a9de-07eb972ada51.png)

if so it is no using the pen directly as far as i understand. the tablet is used as mouse by windows and then the mouse is used in xournalpp which means you are not using the pen itself as input

**Comment by Dartypier** [comment]
This happened also to me, when I messed up with drivers of a Wacom tablet. I uninstalled Xournal++ and reinstalled it, then everything worked as before. 

So try to uninstall Xournal++ completely, the reinstall it, and see if it works.

**Reply by Vulkanodox** [reply]
did that already but did not solve it.

figured it out finally. Had to remove the drivers and xournalpp. delete all the settings for the driver and xournalpp in the appdata folder. Remove every file in the program files directory.

then install the driver again, then install xournalpp. Any other way around and it doesn't work.

**Comment by Vulkanodox** [comment, marked as answer]
figured it out finally. Had to remove the drivers and xournalpp. delete all the settings for the driver and xournalpp in the appdata folder. Remove every file in the program files directory.

then install the driver again, then install xournalpp. Any other way around and it doesn't work.
